kymatio/scattering1d/backend/torch_backend.py

```diff
@@ -132,19 +92,18 @@ def unpad(x, i0, i1):
 
 
 fft = FFT(lambda x: torch.fft(x, 1, normalized=False),
+    lambda x: torch.rfft(x, 1, normalized=False, onesided=False),
     lambda x: torch.ifft(x, 1, normalized=False),
     lambda x: torch.irfft(x, 1, normalized=False, onesided=False),
-    type_checks)
+    contiguous_check, lambda x: None, complex_check)
```

Comment:
why are these in with the fft functions? I'd prefer them to have names, in the backend.

As a matter of fact: why do we even have this fft object nested inside the backend?